.github/workflows/deploy_schema_documentation.yml

```diff
@@ -1,8 +1,8 @@
 name: Deploy Schema Documentation
-on:
-  push:
-    branches:
-      - master
+on: ["push"]
+  #push:
+  #  branches:
+#   - master
```

Comment:
Can these commented lines be removed? No need to check that it is the master branch?

Reply:
Oh, I forgot to push the last commit that changed that back.